bounded-collections/Cargo.toml

```diff
@@ -19,10 +19,12 @@ serde_json = "1.0.41"
 
 [features]
 default = ["std"]
+serde = [
+    "dep:serde"
+]
```

Comment:
```suggestion
```

I just realized that my previous review I have overseen this :P The feature `serde` already exists, because of the optional dependency `serde`.

Reply:
ahh, right. unexplicit, but it is there...